### ISSUE TITLE
sagews: properly display objects of type sage.repl.image.Image

### ISSUE THREAD

**Issue by haraldschilly** [sagemathinc/cocalc#2682]
In a sage worksheet, objects of type `sage.repl.image.Image` should be displayed as such. Here is some code with a workaround:

```
M = mandelbrot_plot(x_center=-.5)
M
---
type(M)
---
M = mandelbrot_plot(x_center=-.5)
M = external_ray([0,0.05,..1], image=M)
fn = 'mandelbrot.png'
M.save(fn)
salvus.file(fn)
```

Link to published worksheet:

**Comment by williamstein** [comment]
One reason it doesn't work is that I don't even know about this sage.repl.image.Image, so definitely never implemented support for it in Sagews.  Thanks for pointing this out...

**Comment by haraldschilly** [comment]
yes, me neither, I figured that's the case. I saw this when I tried to look what these fractal plot functions look like...
